postcode_lookup/locale/cy/LC_MESSAGES/messages.po

```diff
@@ -170,6 +171,14 @@ msgstr ""
 "<h3>Ymgeiswyr</h3> <p>Bydd rhestr o ymgeiswyr yr etholiad hwn yn cael ei "
 "chyhoeddi ar ôl %(sopn_date)s.</p>"
 
+#: postcode_lookup/templates/includes/ballots.html:43
+msgid ""
+"There may also be parish, town or community council elections in some "
+"areas."
+msgstr ""
+"Mae'n bosib y bydd etholiadau cynghorau plwyf, tref neu gymuned mewn rhai"
+" ardaloedd."
```

Comment:
This is the translation we use on the widget

https://github.com/DemocracyClub/WhereDoIVote-Widget/blob/2d32ed1d86c0a49a5969535548d3ebd4937cb824/src/translations/cy.json#L57

Google translate seems to reckon it is right

![Screenshot at 2025-04-09 16-17-16](https://github.com/user-attachments/assets/10b84cee-c658-45e6-8a3a-e4d87f7fd106)